pyirf/statistics.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+
+def li_ma_significance(n_on, n_off, alpha=0.2):
+    '''
+    Calculate the Li & Ma significance.
+
+    Formula (17) doi.org/10.1086/161295
+
+    This functions returns 0 significance when n_on < alpha * n_off
+    instead of the negative sensitivities that would result from naively
+    evaluating the formula.
+
+    Parameters
+    ----------
+    n_on: integer or array like
+        Number of events for the on observations
+    n_off: integer of array like
+        Number of events for the off observations
+    alpha: float
+        Ratio between the on region and the off region size or obstime.
+
+    Returns
+    -------
+    s_lima: float or array
+        The calculated significance
+    '''
+
+    scalar = np.isscalar(n_on)
```

Comment:
why not use gammapy.stats for this rather than implement it again? It's already a dependency for the I/O

Reply:
Where is it used right now?

Anyway we want to get rid of it if possible (apart of plotting)

Reply:
Because we wanted to get rid of gammapy as a dependency. We won't need it for the IO.

Reply:
OK, if you are removing gammapy as a dependency then keep it here.  But I would cross check the implementation with the one there. 

This is needed for both the optimization and for the sensitivity computation (to deal with the  5 sigma minimum per bin requirement), so we do need an implementation.

Reply:
> won't need it for the IO.

You will re-implement all of it and the plotting as well? Seems like duplication of effort

Reply:
I updated issue #21 about this, I think what I propose is a good compromise.

Reply:
What I/O functionality is provided by gammapy?